Kickstarter-iOS/Views/Controllers/LoginToutViewController.swift

```diff
@@ -232,7 +232,7 @@ internal final class LoginToutViewController: UIViewController, MFMailComposeVie
     self.dismiss(animated: true, completion: nil)
   }
 
-  @objc fileprivate func helpButtonPressed() {
+  @IBAction fileprivate func helpButtonPressed() {
```

Comment:
noticed we set the help button programmatically `#selector(helpButtonPressed)`, so is the selector being set in the storyboard and in code?